jb-lib/lib/channel_handler/telegram_handler.py

```diff
@@ -0,0 +1,382 @@
+import base64
+import json
+import os
+from typing import Any, Dict, Generator
+import requests
+
+from sqlalchemy import select
+
+from lib.db_connection import sync_session
+from lib.azure_storage_sync import AzureStorageSync
+from lib.channel_handler.channel_handler import ChannelData, RestChannelHandler, User
+from lib.channel_handler.language import LanguageMapping, LanguageCodes
+from lib.data_models import (
+    MessageType,
+    Message,
+    TextMessage,
+    AudioMessage,
+    InteractiveMessage,
+    Option,
+    FormMessage,
+    ImageMessage,
+    DocumentMessage,
+    InteractiveReplyMessage,
+    RestBotInput,
+    ListMessage,
+    ButtonMessage,
+    DialogMessage,
+    DialogOption,
+    FormReplyMessage
+)
+from lib.models import JBChannel, JBUser, JBForm
+from lib.utils import EncryptionHandler
+
+azure_creds = {
+    "account_url": os.getenv("STORAGE_ACCOUNT_URL"),
+    "account_key": os.getenv("STORAGE_ACCOUNT_KEY"),
+    "container_name": os.getenv("STORAGE_AUDIOFILES_CONTAINER"),
+    "base_path": "input/",
+}
+storage = AzureStorageSync(**azure_creds)
+
+
+class TelegramHandler(RestChannelHandler):
+
+    @classmethod
+    def is_valid_data(cls, data: Dict) -> bool:
+        return "message" in data or "edited_message" in data
```

Comment:
Is there any better way to identify if the message is from telegram?

Reply:
fixed